src/lib/types.ts

```diff
@@ -23,6 +23,7 @@ export type Provider =
 export type AuthChangeEventMFA = 'MFA_CHALLENGE_VERIFIED'
 
 export type AuthChangeEvent =
+  | 'INITIAL_SESSION'
```

Comment:
Add tsdoc somewhere about it

Reply:
will document it in https://supabase.com/docs/reference/javascript/auth-onauthstatechange